src/core/store/modules/collateral/collateral.actions.ts

```diff
@@ -135,15 +167,65 @@ const claimRevenue = createAsyncThunk<{ contract: string; success: boolean }, Cl
   }
 );
 
+const claimOperatorTokens = createAsyncThunk<{ claimed: string | undefined }, ClaimOperatorTokenProps, ThunkAPI>(
+  'collateral/claimOperatorTokens',
+  async (props, { extra }) => {
+    const { collateralService } = extra.services;
+    const tx = await collateralService.claimOperatorTokens(props);
+    return {
+      claimed: tx.value?.toString(),
+    };
+  }
+);
+// TODO: Does there need to be a reducer for this?
```

Comment:
yea can optimistically update state and set operatorTokens to 0

Reply:
tbh that should be how we handle all the optimistic updates, incl positions. table

Reply:
Will do all this in a separate PR